### ISSUE TITLE
[Mozilla] (depends on #9) SQL script to fix `Is_Backed_Out_Changeset` and `Backout_Hashes` for Back-out changesets.

### ISSUE THREAD

**Issue by quocbui2020** [quocbui2020/requirement_description_to_bug_counts#10]
- Depends on #9

**Comment by quocbui2020** [comment]
The SQL script is no longer needed. Read the comment in #9
